### PR TITLE
 :recycle:  Card.Media - introducing  grid, adding Card.Content

### PR DESCRIPTION
**Pull request by laliq**
The grid was created. I =n this stage there is question why `Card` component doesn't have`Card.Content` component?

### CODE REVIEW

File: libraries/core-react/src/components/Card/CardMedia.tsx [hunk 1]
```diff
@@ -37,6 +34,7 @@ const StyledCardMedia = styled.div<CardMediaProps>`
           }
         `
       : css`
+          padding: 0 16px;
```
Comment (mimarz) [inline libraries/core-react/src/components/Card/CardMedia.tsx:37]: Suggest updating this to use the token `spacings` values so its updates correctly if spacings change (the same way you've done in the other components).

File: libraries/core-react/src/components/Card/CardMedia.tsx [hunk 2]
```diff
@@ -11,21 +11,18 @@ export type CardMediaProps = {
 } & HTMLAttributes<HTMLDivElement>
 
 const StyledCardMedia = styled.div<CardMediaProps>`
-  display: flex;
-  width: 100%;
+  display: grid;
```
Comment (mimarz) [inline libraries/core-react/src/components/Card/CardMedia.tsx:14]: nitpicking, but I think `place-items` still works with `display:flex`?

File: libraries/core-react/src/components/Card/CardMedia.tsx [hunk 3]
```diff
@@ -11,21 +11,18 @@ export type CardMediaProps = {
...
+  place-items: center;
   &:last-child {
-    margin-bottom: 8px;
+    padding-bottom: 24px;
```
Comment (mimarz) [inline libraries/core-react/src/components/Card/CardMedia.tsx:17]: Should images still have a bottom spacing of `24px` when they are last in the Card? @equinor/eds-ux

**Review by mimarz — CHANGES_REQUESTED** [review]
Looks very good! 👍 

Just some small nitpicking and cleaning up feedbacks. I think we need to figure out what the correct spacing for the last image should be with @equinor/eds-ux

File: libraries/core-react/src/components/Card/CardMedia.tsx (hunk 2 above)
Comment (laliq) [reply libraries/core-react/src/components/Card/CardMedia.tsx:14]: `width:auto` was added and, `display:flex` is back. It looks that component doesn't need `place-items`

File: libraries/core-react/src/components/Card/CardMedia.tsx (hunk 1 above)
Comment (laliq) [reply libraries/core-react/src/components/Card/CardMedia.tsx]: Good point, done.

**Comment by benteljones** [comment]
https://www.figma.com/file/MDkIHZdkMUw23EKI5HT83T/Bente?node-id=169%3A53764

File: libraries/core-react/src/components/Card/CardMedia.tsx (hunk 3 above)
Comment (mimarz) [reply libraries/core-react/src/components/Card/CardMedia.tsx:17]: This will be looked at in #1713

**Review by mimarz — APPROVED** [review]
Great work! 🎉👍

**Pull request merged**